python_tests/pytest_opinfos.py

```diff
@@ -1103,6 +1105,24 @@ def torch_reshape_sym_fn(input_tensor, output_shaped_tensor):
 )
 tensor_creation_ops.append(uniform_opinfo)
 
+matmul_ops = []
+
+
+def matmul_wrapper(fd, a, b):
+    t1 = fd.ops.matmul(a, b)
+    t2 = fd.ops.cast(t1, dtype=DataType.Half)
+    return t2
+
+
+matmul_opinfo = OpInfo(
```

Comment:
This is not a general matmul test, regarding the output dtype cast. Let's name it properly for that as well.

Reply:
I was I saw this was, this was the general matmul test. When we swap out the aten::matmul for Fuser's implementation, the user will not be aware of that. Please let me know if you think I have that wrong.